problems/removeVowels.js

```diff
@@ -6,7 +6,8 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
+function removeVowels(str) {
+    return str.replace(/[aeiouAEIOU]/g, "")
```

Comment:
This is very very slick. Make sure that you fully understand and know this line of code. YOu should be able to reproduce this answer in an interview. If you don't already actually know regex how would do it without it?